sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/InMemoryCatalog.scala

```diff
@@ -116,7 +116,7 @@ class InMemoryCatalog(
         fs.mkdirs(location)
       } catch {
         case e: IOException =>
-          throw new SparkException(s"Unable to create database ${dbDefinition.name} as failed " +
+          throw new IOException(s"Unable to create database ${dbDefinition.name} as failed " +
```

Comment:
Is it possible users depend on the conversion to SparkException? It seems safer to leave it as is.